docs/architecture/decisions/0003-cross-service-authorisation.md

```diff
@@ -0,0 +1,23 @@
+# 3. Cross-service authorisation
+
+Date: 2024-03-12
+
+## Status
+
+Accepted
+
+## Context
+
+When an LPA is created or updated in the LPA Store, we need to keep a record of who the initiator of the action is. This ensures we have an audit trail of all changes, whether they come from a member of the public, a member of staff at the OPG or an internal service.
+
+## Decision
+
+Requests to create or update an LPA will include a URN identifier to point to the soruce of the action. This will be in the format: `urn:opg:${service}:users:${identifier}`.
```

Comment:
Would this would be in addition to the actorUID? Something like:

```json
{
  "uid": "M-1234-5678-9012",
  "actorType": "attorney",
  "actorUID": "urn:opg:poas:makeregister:users:740e5834-3a29-46b4-9a6f-16142fde533a",
  "accessCode": "abcdef123456",
  "author": "urn:opg:poas:makeregister:users:25cc4507-b381-4e6f-80af-26578fd21ed6"
}
```

`author` may not be the right term here but hopefully you get where I'm coming from? If this is the case we may want to define the expected event field name in the ADR too.